docs/strategy_analysis_example.md

```diff
@@ -250,3 +250,75 @@ fig.show()
 ```
 
 Feel free to submit an issue or Pull Request enhancing this document if you would like to share ideas on how to best analyze the data.
+
+## Analyse the buy/entry and sell/exit tags
```

Comment:
This is the only file you shouldn't put this documentation to.

it's auto-generated from the jupyter notebook via [this method](https://www.freqtrade.io/en/latest/developer/#updating-example-notebooks) ... 
so if you think it belongs here - it should also go to the notebook.
That doesn't make too much sense though - as it's only description - not much code to explore interactively.

 i think you might want to create a new page for this in the data-analysis section instead

Reply:
OK, I'll move it to another new page

Reply:
Moved docs in [b3cb722](https://github.com/freqtrade/freqtrade/pull/6543/commits/b3cb7226467c359b0748cb4866f8f8fcfa649a53)